server/routes/standardRouter.ts

```diff
@@ -1,31 +1,16 @@
 import { Router } from 'express'
-import csurf from 'csurf'
 import auth from '../authentication/auth'
 import tokenVerifier from '../data/tokenVerification'
 import populateCurrentUser from '../middleware/populateCurrentUser'
 import type UserService from '../services/userService'
 import populateCurrentUserRoles from '../middleware/populateCurrentUserRoles'
 
-const testMode = process.env.NODE_ENV === 'test'
-
 export default function standardRouter(userService: UserService): Router {
   const router = Router({ mergeParams: true })
 
   router.use(auth.authenticationMiddleware(tokenVerifier))
   router.use(populateCurrentUser(userService))
   router.use(populateCurrentUserRoles())
 
-  // CSRF protection
```

Comment:
The problem was that CSRF was being setup here in this middleware .... but this middleware is only being called for routes associated with the mail room journey